flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/source/HBaseLookupFunction.java

```diff
@@ -45,6 +47,7 @@
  * The HBaseLookupFunction is a standard user-defined table function, it can be used in tableAPI
  * and also useful for temporal table join plan in SQL.
  */
+@Internal
```

Comment:
Have `testHBaseLookupFunction`, do we need expose this?

Reply:
The `testHBaseLookupFunction` is a temporary solution when this test was introduced, because temporal join is not supported at that time. I will refactor the tests in the next pull request. So I think we don't need to expose this class.